EIPS/eip-2981.md

```diff
@@ -12,33 +12,35 @@ requires: 165
 
 ## Simple Summary
 
-A standardized way to retrieve royalty payment information for ERC-721 tokens to enable universal support for royalty payments in all NFT marketplaces and ecosystem participants.
+A standardized way to retrieve royalty payment information for NFT tokens to enable universal support for royalty payments in all NFT marketplaces and ecosystem participants.
 
 ## Abstract
 
-This standard extends the [ERC-721 specification](./eip-721.md) to enable setting a royalty amount paid to the NFT creator or rights holder every time an NFT is sold and re-sold. This is intended for NFT marketplaces that want to support the ongoing funding of artists and other NFT creators. The royalty payment must be voluntary as required by the EIP-721 standard, as `transferFrom()` includes NFT transfers between wallets, and executing `transferFrom()` does not always imply a sale occurred. Marketplaces and individuals implement this standard by retrieving the royalty payment information with `royaltyInfo()`, which specifies how much to pay to which address for a given sale price. The exact mechanism for paying and notifying the recipient will be defined in future EIPs. This ERC should be considered a minimal, gas-efficient building block for further innovation in NFT royalty payments.
+This standard extends NFT standards such as the [ERC-721](./eip-721.md) and [ERC-1155]](./eip-1155.md) specifications to enable setting a royalty amount paid to the NFT creator or rights holder every time an NFT is sold and re-sold. This is intended for NFT marketplaces that want to support the ongoing funding of artists and other NFT creators. The royalty payment must be voluntary, as `transferFrom()` includes NFT transfers between wallets, and executing `transferFrom()` does not always imply a sale occurred. Marketplaces and individuals implement this standard by retrieving the royalty payment information with `royaltyInfo()`, which specifies how much to pay to which address for a given sale price. The exact mechanism for paying and notifying the recipient will be defined in future EIPs. This ERC should be considered a minimal, gas-efficient building block for further innovation in NFT royalty payments.
```

Comment:
> 'The royalty payment must be voluntary, as `transferFrom()` includes NFT transfers between wallets, and executing `transferFrom()` does not always imply a sale occurred.'

Suggest: Remove references to transferFrom() and mention that enforced royalty payments can always be defeated by private arrangements.

Rationale:
Given that we are loosening the dependency on existing NFT standards, I think including specific transfer APIs is unnecessary and misleading. This section also does not address the more compelling argument that enforcing the royalty in a contract is actually impossible because extra payments can happen through a private arrangement.